examples/VisualNovel/portrait_manager_ex.gd

```diff
@@ -0,0 +1,127 @@
+extends PortraitManager
```

Comment:
Why this extention is a duplicate of the original script?

Reply:
Wdym duplicate? The script is different because the original script does not support non-TextureRect portraits